shared/src/global.d.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Same here, change eslint rules? @bibixx

Reply:
That is kind of expected and an exception to the rule, where we _generally_ shouldn't use `Date` in the shared folder, but it sometimes is necessary.